library/CHART-TEMPLATE/tests/common_library_labels_test.yaml

```diff
@@ -0,0 +1,44 @@
+suite: test labels helpers
+templates:
+  - templates/deployment.yaml # we could check the same for service and serviceAccount as well
+release:
+  name: my-release
+  namespace: my-namespace
+set:
+  licenseKey: test-license-key
+  cluster: test-cluster
+tests:
+  # We cannot test behaviour without overwriting this since the version chart often changes
+  - it: sets labels according to values
+    set:
+      labels:
+        helm.sh/chart: fixed
```

Comment:
```suggestion
  - it: sets labels according to values
    set:
      labels:
        helm.sh/chart: fixed  # We cannot test behavior without overwriting this since the version chart often changes
```